src/common/utils/kaltura-params.ts

```diff
@@ -125,6 +125,15 @@ function getReferrer(): string {
   return referrer;
 }
 
+function isInUnfriendlyIframe() {
+  try {
+    window.parent.document.URL;
```

Comment:
i dont really like a line that (on first glance) does nothing
how about this ?

try {
    return window.self.location.href !== window.top.location.href;
} catch() {
    return true;
}